pydantic/mypy.py

```diff
@@ -68,7 +68,8 @@
 BASESETTINGS_FULLNAME = 'pydantic.env_settings.BaseSettings'
 FIELD_FULLNAME = 'pydantic.fields.Field'
 DATACLASS_FULLNAME = 'pydantic.dataclasses.dataclass'
-BUILTINS_NAME = 'builtins' if float(mypy_version) >= 0.930 else '__builtins__'
+MYPY_VERSION = tuple(int(part) for part in mypy_version.split('+')[0].split('.'))
+BUILTINS_NAME = 'builtins' if MYPY_VERSION >= (0, 930) else '__builtins__'
 
 
 def plugin(version: str) -> 'TypingType[Plugin]':
```

Comment:
You can use `version` string here as well.

Reply:
What benefit would that provide over importing it directly?

Reply:
Something like this?
```py
BUILTINS_NAME: str


def plugin(version: str) -> 'TypingType[Plugin]':
    global BUILTINS_NAME
    BUILTINS_NAME = tuple(int(part) for part in version.split('+')[0].split('.'))

    return PydanticPlugin
```